lib/cardconnect/services/authorization/authorization_response.rb

```diff
@@ -4,7 +4,7 @@ class AuthorizationResponse
       include Utils
 
       FIELDS = [:respstat, :retref, :account, :token, :amount, :merchid, :respcode,
-                :resptext, :respproc, :avsresp, :cvvresp, :authcode, :commcard, :profileid]
+                :resptext, :respproc, :avsresp, :cvvresp, :authcode, :commcard, :profileid, :acctid]
```

Comment:
Does this field come over when getting an Auth response from CardConnect's API?  It does not appear to be in their documentation unless I am missing something.